recipes-omnect/initrdscripts/omnect-os-initramfs/flash-mode-2

```diff
@@ -33,7 +34,7 @@ flash_mode_2_enabled() {
 run_bmap_tool() {
     local umount_dev=""
     local bmap_ret=""
-
+    
```

Comment:
please check your editor settings to trim whitespaces at eol

Reply:
done